### PR TITLE
Add catserver/foxserver

### PR DESCRIPTION
**Pull request by Soapy7261**
Adds catserver/foxserver (same devs, foxserver was merged into catserver after being DMCA'd), also adds a slight correction to mohist, i mean nobody should talk down a project just for it being in this list but you know, there are people like that.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,10 +21,11 @@ A non-exhaustive list of projects to be known to have rather unpleasant vibes.
   & [Mirai](https://github.com/etil2jz/Mirai), "The combined power of Paper forks for maximum instability!" - Tossed
   salad of people that haven't even really understood the patch system. And
   yes, *[it's that bad](https://github.com/kennytv/Yaptapia)*
+* [CatServer](https://github.com/Luohuayu/CatServer)/[FoxServer](https://github.com/Luohuayu/FoxServer), "Steals code from other hybirds and breaks their license" - Gets angry when mohist and magma DMCA them for [breaking](https://github.com/Luohuayu/FoxServer/issues/7#issuecomment-1100009357) their license
 * [Mohist](https://github.com/MohistMC/Mohist), "Get the worst parts of Bukkit and Forge and mash them together"
-  - [Swapped Essentials' plugin jar](https://github.com/MohistMC/Mohist/blob/70a303f4d02e9480cc5472c1c26f7d9cb6560732/src/fmllauncher/java/com/mohistmc/AutoDeletePlugins.java#L20-L22)
+  - (in the past) [Swapped Essentials' plugin jar](https://github.com/MohistMC/Mohist/blob/70a303f4d02e9480cc5472c1c26f7d9cb6560732/src/fmllauncher/java/com/mohistmc/AutoDeletePlugins.java#L20-L22)
```
Comment (Andre601) [inline README.md:26]: Not really needed. "Swapped" already has a past connatation (was it connatation?) so this addition is somewhat pointless...

Comment (Soapy7261) [reply README.md]: oops

**Comment by Andre601** [comment]
To be honest, wouldn't it be a good idea to replace mohist with a general "Hybrid servers (Forge/something else + bukkit)" list and then point out actual bad actors individually?

Like as an example:
```markdown
* Hybrid Servers (Forge/Something else + Bukkit) "Smash two separate APIs together into a mess" - Servers that combine two incompatible APIs and break them both in the process.
  * [Mohist](https://github.com/MohistMC/Mohist) was known for [swapping out EssentialsX's plugin jars](https://github.com/MohistMC/Mohist/blob/70a303f4d02e9480cc5472c1c26f7d9cb6560732/src/fmllauncher/java/com/mohistmc/AutoDeletePlugins.java#L20-L22) after a [submitted PR](https://github.com/EssentialsX/Essentials/pull/3580) to fix their own mistake was rejected. *Then* tried to turn [EssentialsX's PSA](https://essentialsx.net/do-not-use-mohist.html) into [an ad](https://github.com/EssentialsX/Website/pull/44)
  * [CatServer](https://github.com/Luohuayu/CatServer)/[FoxServer](https://github.com/Luohuayu/FoxServer) "[Insert 'I made this' meme here]" - Devs got mad after Mohist and Magma filed DMCA claims for [breaking their licenses](https://github.com/Luohuayu/FoxServer/issues/7#issuecomment-1100009357).
```
To be honest, it feels a bit messy. It's up to you to decide.

**Comment by Soapy7261** [comment]
I mean, I'd make the wording a little less discrediting (it's not easy making a hybird) but I'll see how i can add that

**Pull request merged**